### PR TITLE
Implemented Logit-Normal Distribution

### PR DESCRIPTION
**Pull request by ks147**
* stats
  * Implemented Logit-Normal probability distribution

### CODE REVIEW

**Comment by abh2k** [comment]
@czgdp1807 please review this pr.

File: sympy/stats/crv_types.py [hunk 1]
```diff
@@ -2449,6 +2451,93 @@ def LogLogistic(name, alpha, beta):
...
+#-------------------------------------------------------------------------------
+#Logit-Normal distribution------------------------------------------------------
+
+class LogitNormalDistribution(SingleContinuousDistribution):
+    _argnames = ('mu', 's')
+    set = Interval(0, 1)
+
+    @staticmethod
+    def check(mu, s):
+        _value_check(s**2 > 0, "Scale parameter s must be positive.")
```
Comment (czgdp1807) [inline sympy/stats/crv_types.py:2463]:
```suggestion
        _value_check((s**2).is_real is not False and s**2 > 0, "Squared scale parameter s must be positive.")
        _value_check(mu.is_real is not False, "Location parameter must be real")
```

File: sympy/stats/crv_types.py [hunk 2]
```diff
@@ -2449,6 +2451,93 @@ def LogLogistic(name, alpha, beta):
...
+
+    def logit(self, x):
```
Comment (czgdp1807) [inline sympy/stats/crv_types.py:2465]: Shouldn't it be private?
```suggestion
    def _logit(self, x):
```

File: sympy/stats/crv_types.py [hunk 3]
```diff
@@ -2449,6 +2451,93 @@ def LogLogistic(name, alpha, beta):
...
+    def pdf(self, x):
+        mu, s = self.mu, self.s
+        return exp(-(self.logit(x) - mu)**2/(2*s**2))*(S.One/sqrt(2*pi*(s**2)))*(1/(x*(1 - x)))
+
+    def _cdf(self, x):
+        mu, s = self.mu, self.s
+        return (S.One/2)*(1 + erf((self.logit(x) - mu)/(sqrt(2*s**2))))
+
+    def _moment_generating_function(self, t):
+        print("No analytical solution")
```
Comment (czgdp1807) [inline sympy/stats/crv_types.py:2477]:
```suggestion
```
If no analytical solution is available, do not define that method in the subclass.

File: sympy/stats/crv_types.py [hunk 4]
```diff
@@ -2449,6 +2451,93 @@ def LogLogistic(name, alpha, beta):
 
     return rv(name, LogLogisticDistribution, (alpha, beta))
 
+#-------------------------------------------------------------------------------
+#Logit-Normal distribution------------------------------------------------------
+
+class LogitNormalDistribution(SingleContinuousDistribution):
+    _argnames = ('mu', 's')
+    set = Interval(0, 1)
```
Comment (czgdp1807) [inline sympy/stats/crv_types.py:2459]:
```suggestion
    set = Interval.open(0, 1)
```
I believe `logit` isn't defined for `x = 0` and `x = 1`.

File: sympy/stats/crv_types.py (hunk 2 above)
Comment (ks147) [reply sympy/stats/crv_types.py]: Yeah I've made the required changes in the latest commit

**Comment by abh2k** [comment]
@czgdp1807 @ks147  any updates.

**Comment by ks147** [comment]
> @czgdp1807 @ks147 any updates.

Yeah I made the required changes some time ago and commited them

**Comment by czgdp1807** [comment]
LGTM. Will merge tomorrow if no objections raised.

**Pull request merged**